code/__HELPERS/_macros.dm

```diff
@@ -131,6 +133,10 @@
 
 #define isatom(A) (istype(A, /atom))
 
+#define issqueletic(A) (isskellington(A) || isskelevox(A)) //Yes, it is a word
```

Comment:
I'm not entirely sure 'squeletic' is something I'd ever use to describe anything ever
Why not skelletal?